src/languages/es.ts

```diff
@@ -2246,6 +2248,43 @@ const translations = {
         byAddingThisBankAccount: 'Al añadir esta cuenta bancaria, confirmas que has leído, comprendido y aceptado',
         owners: 'Dueños',
     },
+    ownershipInfoStep: {
```

Comment:
https://expensify.slack.com/archives/C01GTK53T8Q/p1731423389064609
Getting a confirmation for the translation